### PR TITLE
Update npm dependencies

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Migrating [this PR](https://github.com/openshift/origin/pull/8670) from origin to the new web console repo.

### CODE REVIEW

**Comment by jwforres** [comment]
[merge]

File: Gruntfile.js [hunk 1]
```diff
@@ -201,19 +202,21 @@ module.exports = function (grunt) {
...
           'bower_components/uri.js/src/URI.fragmentQuery.js',
+          // bower registration error? we get 2x versions of uri.js/urijs
+          'bower_components/urijs/',
           'bower_components/messenger/build/css/messenger.css',
           'bower_components/messenger/build/css/messenger-theme-future.css',
           'bower_components/messenger/build/css/messenger-theme-block.css',
           'bower_components/messenger/build/css/messenger-theme-air.css',
           'bower_components/messenger/build/css/messenger-theme-ice.css',
           'bower_components/messenger/build/js/messenger-theme-future.js',
-          'bower_components/moment-timezone/builds/moment-timezone-with-data-2010-2020.js',
```
Comment (benjaminapetersen) [inline Gruntfile.js]: This is exclusion is new since my original PR.

File: Gruntfile.js [hunk 2]
```diff
@@ -224,7 +227,6 @@ module.exports = function (grunt) {
           'bower_components/datatables/media/css/jquery.dataTables.css',
           'bower_components/datatables-colreorder/css/dataTables.colReorder.css',
           'bower_components/datatables-colvis/css/dataTables.colVis.css',
-          'bower_components/eonasdan-bootstrap-datetimepicker/build/js/bootstrap-datetimepicker.min.js',
```
Comment (benjaminapetersen) [inline Gruntfile.js]: This one is also new.  I should put these back into the file.

**Comment by benjaminapetersen** [comment]
@jwforres 2 comments. Sorry, hadn't completely finished my line-by-line review of this one before the merge!  Fixing.

**Comment by jwforres** [comment]
i killed the merge, let me know when you are ready to have me look at it again

**Comment by benjaminapetersen** [comment]
@jwforres OK! reconcile complete, verified all the changes match the previous PR in origin, + updates since my PR opened.  should be good to go.

**Comment by jwforres** [comment]
[merge]

**Comment by jwforres** [comment]
Noticed this same issue earlier when I tried to merge this, it's getting
stuck trying to launch phantomjs, seems like some package update is
conflicting with the spec tests. Do the tests work when you run them
locally?

**Comment by benjaminapetersen** [comment]
@jwforres test run successful locally, however I have to manually trust certs in safari before chrome loads & runs the tests.  I'm suspicious its a similar issue...

**Comment by jwforres** [comment]
weird this would decide to pop up now, we've always had the self-signed cert for the tests.  unless it used to launch chrome in one mode and now its launching it in a different mode
trying [test] again

**Comment by jwforres** [comment]
The place where it is getting stuck are spec tests from the output it doesn't look like it launches an HTTPS server for those anyway so I don't think the cert is the issue

**Comment by jwforres** [comment]
I would say either the karma changes are to blame or the serveStatic changes.  Try taking out the karma related bumps and see if it will pass in jenkins

**Comment by benjaminapetersen** [comment]
Will do

**Comment by benjaminapetersen** [comment]
[TEST]

**Comment by benjaminapetersen** [comment]
[test]

**Comment by jwforres** [comment]
failure was the one from earlier today, it was hung up and finally timed
out [test] ben's new changes now

**Comment by benjaminapetersen** [comment]
Oi.

**Comment by jwforres** [comment]
@benjaminapetersen yeah the latest job is stuck too, so looks like serveStatic is not to blame, im gonna blame the karma updates as the likely culprit

**Comment by benjaminapetersen** [comment]
Messing with them now 😒

**Comment by jwforres** [comment]
went ahead and just killed the jenkins job that was stuck

**Comment by benjaminapetersen** [comment]
aaaand [TEST] some more.

**Comment by jwforres** [comment]
Well now we know the culprit :-)

Going to go ahead and [merge] this, we can worry about trying to bump the karma deps later, since karma is only used for running the spec tests and not part of our build chain deps.

**Comment by benjaminapetersen** [comment]
Yay, I'll tinker with Karma later perhaps, after I rebase the bower updates (once back in office).

**Pull request merged**